### PR TITLE
Create sma.rb  Security Management Appliance CISCO

### PR DESCRIPTION
**Pull request by cd67-usrt**

### CODE REVIEW

File: lib/oxidized/model/sma.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+class SMA < Oxidized::Model
+
+	# ESA prompt "mail.example.com> "
+	prompt /^\r*([-\w. ]+\.[-\w. ]+\.[-\w. ]+[#>]\s+)$/
```
Comment (casdr) [inline lib/oxidized/model/sma.rb]: Maybe `\r*([-\.\w]+[#>]\s+)$` is better

**Comment by ytti** [comment]
You need to update README.md too about new models. Is the OS called SMA? If not and you can't find the OS name. Maybe call the model CiscoSMA to avoid stealing potentially 'valuable' TLA, which might be name of an actual OS.

File: lib/oxidized/model/sma.rb (hunk 1 above)
Comment (cd67-usrt) [inline lib/oxidized/model/sma.rb]: capture
No.
match on config file

**Comment by ytti** [comment]
you need to also update README.md, to include the new model. Thanks!

File: lib/oxidized/model/sma.rb (hunk 1 above)
Comment (casdr) [inline lib/oxidized/model/sma.rb]: Whoops, sorry :)

**Pull request merged**